contrib/builder/deb/aarch64/generate.sh

```diff
@@ -97,18 +99,31 @@ for version in "${versions[@]}"; do
 			;;
 	esac
 
-	echo "# Install Go" >> "$version/Dockerfile"
-	echo "# aarch64 doesn't have official go binaries, so use the version of go installed from" >> "$version/Dockerfile"
-	echo "# the image to build go from source." >> "$version/Dockerfile"
+	case "$suite" in
+		jessie)
+			echo "# Install Go (hack: for now use pre-compiled tarball from Hypriot)" >> "$version/Dockerfile"
```

Comment:
If someone has the right code snipped how to build go completely from source I'd be happy to include it.